blocks/embed/embed.js

```diff
@@ -0,0 +1,90 @@
+const loadScript = (url, callback, type) => {
+  const head = document.querySelector('head');
+  const script = document.createElement('script');
+  script.src = url;
+  if (type) {
+    script.setAttribute('type', type);
+  }
+  head.append(script);
+  script.onload = callback;
+  return script;
+};
+
+const getDefaultEmbed = (url) => `<div style="left: 0; width: 100%; height: 0; position: relative; padding-bottom: 56.25%;">
+    <iframe src="${url.href}" style="border: 0; top: 0; left: 0; width: 100%; height: 100%; position: absolute;" allowfullscreen=""
```

Comment:
doesn't this need escaping?

i.e. wouldn't it be possible to create a dom-snippet and then then get the `outerHTML` ?

Reply:
it does need escaping... i added it thanks for the report.

i think that makes it very hard to read, and since these embed snippets come in HTML strings, this blueprint will make it a easier for people to extend the block with their own embeds.